core/packages/PackageInfo.h

```diff
@@ -278,3 +278,4 @@ class PackageInfo {
 
 } // namespace sorbet::core::packages
 #endif
+minTypedLevel_minTypedminTypedLevel_
```

Comment:
What is this doing here?

Reply:
I think that my language client's `textDocument/rename` interacts poorly when the rename operation spans multiple files.